DESCRIPTION

```diff
@@ -1,6 +1,6 @@
-Package: xflow
+Package: recodeflow
 Type: Package
-Title: Converting xFlow Documents to PMML
+Title: Recode variable and variable details to PMML
```

Comment:
The title should probably be,
Contains functions to interface with variables details sheets including recoding variables and convverting them to PMML